lua/lspcontainers/init.lua

```diff
@@ -1,20 +1,22 @@
+local supported_languages = {
+  sumneko_lua = "lspcontainers/lua-language-server:1.20.5",
+  dockerls = "lspcontainers/docker-langserver:0.4.1"
+}
+
 local function command(server, user_opts)
   local opts = user_opts or {}
+
   local workdir = opts.root_dir or vim.fn.getcwd()
-  local image = ""
   local volume = workdir..":"..workdir
 
-  -- TODO: dockerfile exists, needs implementation
-  --if server == "dockerls" then
-    --image = "lspcontainers/docker-langserver:0.4.1"
-  --end
-
-  if server == "sumneko_lua" then
-    image = "lspcontainers/lua-language-server:1.20.5"
-  end
+  local additional_languages = opts.additional_languages or {}
```

Comment:
Question - what happens in the situation when a use provides invalid servers to the configuration? An example of this would be anything that doesn't exist in the `lspconfig` repo as that is essentially what we directly support.

My understanding by looking at the code would be it would successfully register in `lspcontainers` but then fail when trying to run `setup` in `lspconfig`. Is this desired? Should we verify that its a proper server in our logic before then?

Reply:
![image](https://user-images.githubusercontent.com/3665200/117391415-821a3000-aeb5-11eb-8d43-8e922c5bc3a3.png)

Reply:
From @adrianord in Twitch chat: 

> Okay, so lspcontainer and LspConfig are decoupled right now, so it is the responsibility of the end user to ensure that their configuration is correct. LspConfig lets you add your own languages through its own API so they should do so if they need to.